wpilibcIntegrationTests/include/command/MockCommand.h

```diff
@@ -11,12 +11,12 @@
 
 class MockCommand : public Command {
  private:
-  int m_initializeCount;
-  int m_executeCount;
-  int m_isFinishedCount;
+  int32_t m_initializeCount;
```

Comment:
The usage of int only applied to user-facing interfaces. Tests aren't user-facing, so I changed the types.